sematic/api_client.py

```diff
@@ -237,6 +237,38 @@ def get_external_resource(
     return ExternalResource.from_json_encodable(response["external_resource"]).resource
 
 
+def activate_external_resource(resource_id: str) -> AbstractExternalResource:
+    """Activate the external resource on the server, return the result.
+
+    Parameters
+    ----------
+    resource_id:
+        The id of the resource to activate.
+
+    Returns
+    -------
+    The resource as saved by the server.
+    """
+    response = _post(f"/external_resources/{resource_id}/activate", json_payload={})
+    return ExternalResource.from_json_encodable(response["record"]).resource
```

Comment:
On error, the response will contain "message" instead of "record".

Reply:
In the error case, the code won't make it to this line. It will raise from the `_post` due to [`_raise_for_response`](https://github.com/sematic-ai/sematic/blob/main/sematic/api_client.py#L518)